frontend/public/components/utils/documentation.tsx

```diff
@@ -1,38 +1,18 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
 
-import { productName } from '../../branding';
 import { ExternalLink } from '../utils';
 
 // Prefer the documentation base URL passed as a flag, but fall back to the latest docs if none was specified.
 export const openshiftHelpBase = (window as any).SERVER_FLAGS.documentationBaseURL || 'https://docs.okd.io/latest/';
 
-/* global
-  HELP_TOPICS: false,
-  GET_STARTED_CLI: false,
-  NETWORK_POLICY_GUIDE: false,
-  COMPUTE_RESOURCES_QUOTA: false,
- */
-export enum HELP_TOPICS {
-  GET_STARTED_CLI = 'cli_reference/get_started_cli.html',
-  NETWORK_POLICY_GUIDE = 'admin_guide/managing_networking.html#admin-guide-networking-networkpolicy',
-  COMPUTE_RESOURCES_QUOTA = 'dev_guide/compute_resources.html#dev-compute-resources',
```

Comment:
You have to update everywhere these are used as well

Reply:
All set.